providers/apache/beam/src/airflow/providers/apache/beam/operators/beam.py

```diff
@@ -399,6 +399,10 @@ def execute(self, context: Context):
         if not self.beam_hook:
             raise AirflowException("Beam hook is not defined.")
 
+        if self.runner.lower() == BeamRunnerType.DirectRunner.lower():
```

Comment:
@olegkachur-e what do you think about updating this `if-cause` and disabling these links for all runners except Dataflow? Because as I see in documentation we have a lot of runners https://beam.apache.org/documentation/runners/capability-matrix/. And, also, I think we support all of them with Beam Operator https://github.com/apache/airflow/blob/main/providers/apache/beam/src/airflow/providers/apache/beam/hooks/beam.py#L58C1-L74C1